themes/lambda-theme

```diff
@@ -0,0 +1 @@
+Subproject commit 8d156c26f3b5574d19123c86cca7c3791389d0d6
```

Comment:
The actual theme needs to be in this PR, not the markdown files for testing. We can add a folder called `tests` and symlink the parent folder as `themes/lambda-theme`.

For the time being dump the contents of `themes/lambda-theme` here and move these markdown files to `tests` folder.